### ISSUE TITLE
Single quotation mark is not accepted during Java files generation

### ISSUE THREAD

**Issue by qichuan** [meituan/Firefly#3]
Exception is thrown if the string value is quoted with single quotation mark, please see error stack below.

```
[error] (run-main-0) com.meituan.firefly.ParseException: [42.32] failure: `{' expected but `'' found
[error] 
[error]     3: optional string sound = 'default';
[error] 
[error]                                ^
com.meituan.firefly.ParseException: [42.32] failure: `{' expected but `'' found

    3: optional string sound = 'default';

                               ^
    at com.meituan.firefly.ThriftParser.parseFile(ThriftParser.scala:240)
    at com.meituan.firefly.Compiler$$anonfun$run$1.apply(Compiler.scala:20)
    at com.meituan.firefly.Compiler$$anonfun$run$1.apply(Compiler.scala:19)
    at scala.collection.immutable.List.foreach(List.scala:381)
    at com.meituan.firefly.Compiler.run(Compiler.scala:18)
    at com.meituan.firefly.Main$$anonfun$main$1.apply(Main.scala:13)
    at com.meituan.firefly.Main$$anonfun$main$1.apply(Main.scala:13)
    at scala.Option.foreach(Option.scala:257)
    at com.meituan.firefly.Main$.main(Main.scala:13)
    at com.meituan.firefly.Main.main(Main.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
```

And the Java file generation will be successful if we use double quotation mark 
i.e., `3: optional string sound = "default";`

**Comment by Ponyets** [comment]
We will fix it in next release.

**Comment by Ponyets** [comment]
fixed in 0.2.4
